src/configuration.ts

```diff
@@ -96,7 +97,7 @@ export class Configuration {
 
     public subtasks: object = null;
 
-    public toolbox_map = new Map<AllowedToolboxItem, any>([
+    public toolbox_map = new Map<AllowedToolboxItem, new (..._) => ToolboxItem>([
         [AllowedToolboxItem.ModeSelect, ModeSelectionToolboxItem],
         [AllowedToolboxItem.ZoomPan, ZoomPanToolboxItem],
         [AllowedToolboxItem.AnnotationResize, AnnotationResizeItem],
```

Comment:
`typeof ToolboxItem` doesn't work, so I added an [abstract construct signature](https://www.typescriptlang.org/docs/handbook/2/classes.html#abstract-construct-signatures), which needed a `..._` to match the fact that we have different constructors for different toolbox items. 

Not the prettiest code in the world